src/common/utility.c

```diff
@@ -447,7 +447,7 @@ static cairo_surface_t *_util_get_svg_img(gchar *logo, const float size)
     else
     {
       cairo_t *cr = cairo_create(surface);
-      cairo_scale(cr, factor, factor);
+      cairo_scale(cr, factor/ppd, factor/ppd);
```

Comment:
Hum a bit strange as the height/width is already taking into account the PPD, see some line above:

```
    const float final_width = dimension.width * factor * ppd,
                final_height = dimension.height * factor * ppd;
```